static/js/click_handlers.js

```diff
@@ -889,38 +889,6 @@ export function initialize() {
     // Don't focus links on context menu.
     $("body").on("contextmenu", "a", (e) => e.target.blur());
 
-    // HOTSPOTS
-
-    // open
-    $("body").on("click", ".hotspot-icon", function (e) {
-        // hide icon
-        hotspots.close_hotspot_icon(this);
-
-        // show popover
-        const [, hotspot_name] = /^hotspot_(.*)_icon$/.exec(
-            $(e.target).closest(".hotspot-icon").attr("id"),
-        );
-        const overlay_name = "hotspot_" + hotspot_name + "_overlay";
-
-        overlays.open_overlay({
-            name: overlay_name,
-            $overlay: $(`#${CSS.escape(overlay_name)}`),
-            on_close: function () {
-                // close popover
-                $(this).css({display: "block"});
-                $(this).animate(
-                    {opacity: 1},
-                    {
-                        duration: 300,
-                    },
-                );
-            }.bind(this),
-        });
-
-        e.preventDefault();
-        e.stopPropagation();
-    });
-
     // confirm
     $("body").on("click", ".hotspot.overlay .hotspot-confirm", function (e) {
```

Comment:
We should probably move the next few functions too -- these 3 handlers are the rest of the `hotpots` module's click handlers...